js/content/community.js

```diff
@@ -3029,6 +3029,61 @@ let FriendsPageClass = (function(){
     return FriendsPageClass;
 })();
 
+let GroupsPageClass = (function(){
+
+    function GroupsPageClass() {
+        this.addSort();
+    }
+
+    GroupsPageClass.prototype.addSort = function() {
+        let groups = Array.from(document.querySelectorAll(".group_block"));
+        if (groups.length === 0) { return; }
+
+        groups.forEach((group, i) => group.dataset.esSortdefault = i);
+
+        function sortGroups(sortBy, reversed) {
+            for (let group of groups) {
+                let name = group.querySelector(".groupTitle > a").textContent;
+                let membercount = Number(group.querySelector(".memberRow > a").textContent.match(/\d+/g).join(""));
+                group.dataset.esSortnames = name;
+                group.dataset.esSortmembers = membercount;
+            }
```

Comment:
You only need to run that once

Reply:
You should fix that in your other code too then.

Reply:
I don't think that I'm running anything multiple times in the sorting function, could you give me a reference?

Reply:
Well [here](https://github.com/tfedor/AugmentedSteam/blob/develop/js/content/community.js#L2890) you do this every time you sort, but I get it. It's to update with current information, which made no sense for me, as I don't acquire new information. I didn't pay enough attention to see that initially.

Reply:
In the example you provided this is only run once, since the if-clause checks `friendsFetched`